packages/components/date-picker/__tests__/date-time-picker.test.tsx

```diff
@@ -63,24 +47,18 @@ describe('Datetime Picker', () => {
     // both input shows correct value
     expect((dateInput as HTMLInputElement).value).toBe('2018/03/05')
     expect((timeInput as HTMLInputElement).value).toBe('10:15 AM')
-    wrapper.setProps({
-      format: 'MM-DD-YYYY HH a',
-    })
+
+    format.value = 'MM-DD-YYYY HH a'
```

Comment:
**Suggestion**: line39 - line54

Maybe we can specify the type when defining the variable without using type assertions everywhere.

```ts
const dateInput: HTMLInputElement = document.querySelector(
  '.el-date-picker__time-header > span:nth-child(1) input'
)!
const timeInput: HTMLInputElement = document.querySelector(
  '.el-date-picker__time-header > span:nth-child(2) input'
)!
timeInput.focus()
await nextTick()
// both input shows correct value
expect(dateInput.value).toBe('2018/03/05')
expect(timeInput.value).toBe('10:15 AM')

format.value = 'MM-DD-YYYY HH a'
await nextTick()
expect(dateInput.value).toBe('03-05-2018')
expect(timeInput.value).toBe('10 am')
```

Same in other places.